pyresample/geometry.py

```diff
@@ -1972,6 +1972,19 @@ def _cartopy_proj_params(self):
             return "EPSG:{}".format(self.crs.to_epsg())
         return self.crs.to_proj4()
 
+    def to_odc_geobox(self):
+        """Convert AreaDefinition to ODC GeoBox.
+
+        See: https://odc-geo.readthedocs.io/en/latest/
+        """
+        try:
+            from odc.geo.geobox import GeoBox
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError("Please install 'odc-geo' to use this method.")
+
+        return GeoBox.from_bbox(bbox=self.area_extent, crs=self.crs,
+                                resolution=np.mean([self.pixel_size_x, self.pixel_size_y]), tight=True)
```

Comment:
I think resolution can be a 2 element pair using the `Resolution` class:

https://odc-geo.readthedocs.io/en/latest/_api/odc.geo.Resolution.html#odc.geo.Resolution

Reply:
I know I can't really remember why I didn't use it but will check and change it eventually.